example/server/static/sep24_scripts/google_analytics.js

```diff
@@ -0,0 +1,6 @@
+window.dataLayer = window.dataLayer || [];
```

Comment:
What is this? If this was just a test can we delete it?

Reply:
This was there from before as part of that google_analytics script